src/app/authentication/authentication-routing.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+import { AuthenticationComponent } from './authentication.component';
+
+const routes: Routes = [{ path: '', component: AuthenticationComponent }];
```

Comment:
the component should LoginComponent, so that once the person logs in to this path the login component shws